lib/paranoia.rb

```diff
@@ -232,6 +234,7 @@ def self.acts_as_paranoid(options={})
       alias_method :destroy_without_paranoia, :destroy
 
       include Paranoia
+      include ParanoiaHelper
```

Comment:
@BenMorganIO : I can try to implement this, but, given that this gem is anyway dependent on `rails >= 4.0`, would this make sense?